apps/prairielearn/src/sync/syncFromDisk.ts

```diff
@@ -47,9 +78,21 @@ export async function syncDiskToSqlWithLock(
 
   logger.info('Loading info.json files from course repository');
 
-  const courseData = await timed('Loaded course data from disk', () =>
-    courseDB.loadFullCourse(courseId, courseDir),
+  if (typeof courseData === 'undefined') {
+    courseData = await timed('Loaded course data from disk', () =>
+      courseDB.loadFullCourse(courseId, courseDir),
+    );
```

Comment:
Note to self: the changes here mean that we lose visibility into timing information for loading files from disk during the happy path. I won't request a change in this PR, but it'd be nice to have `timed` be available in other contexts so that we can use this in the _other_ place where we now call `loadFullCourse`.